### DISCUSSION TITLE
Create a character with the login ID

### DISCUSSION THREAD

**Discussion by carolandreazza** [naverz/zepeto-studio-kor#474] [Issue Report (Legacy)]
I created a world in Unity and got the following error:

1. When creating a character in World, you must create a character with the login ID.
- https://github.com/naverz/zepeto-multiplay-example
- https://github.com/naverz/zepeto-multiplay-example/blob/main/Assets/ZepetoScripts/ClientStarter.ts

Does anyone know what it means?

**Comment by Zepeto-TechEvangelist** [comment]
Hello. Did you set up the basic world and log in?  https://studio.zepeto.me/guides/preparing-for-world-creating
And did you add the code to load the characters? https://studio.zepeto.me/guides/character-control
If a problem occurs after that, please upload it with detailed error screen capture.

**Reply by carolandreazza** [reply]
Hello! Thank you for your answer!
But I already followed these guides and still with the error :/
![image](https://user-images.githubusercontent.com/29384128/157660022-6a5b3543-680f-4d92-82ab-0dfaf7ef703b.png)

**Reply by Zepeto-TechEvangelist** [reply]
Did you create your own Zepeto character?
It seems to be a problem because there is no character in the Zepeto app.

**Reply by carolandreazza** [reply]
Would be the ZepetoPlayers? 
Of this guide https://studio.zepeto.me/guides/character-control
If so, i already did.

**Reply by Zepeto-TechEvangelist** [reply]
Can you turn on the Zepeto app, click the profile menu, and show me your ID and character screenshot?
If you fail even though you have an account normally, you should ask the our management part.

![KakaoTalk_Photo_2022-03-14-10-26-52](https://user-images.githubusercontent.com/100118650/158090484-533f5bd8-9cc6-4082-a54f-d691ee4ce5fd.png)

**Reply by carolandreazza** [reply]
Hi!
Unfortunately I still have the same error. Here's the print you asked for:
How do I contact the management part?

![Arquivo_000](https://user-images.githubusercontent.com/29384128/158156384-f702c6c2-2ccb-41e7-9e61-580fe990a797.png)

**Reply by Zepeto-TechEvangelist** [reply]
I'll talk to the person in charge

**Reply by Zepeto-TechEvangelist** [reply]
As a result of checking the person in charge, the error message means that the examination was rejected because there were two problems with the world you submitted.
1. When entering the world, characters without hair and clothes are loaded, not normal login characters. This code needs to be modified.
2. The copyright violation image is in the world. Please delete the image.

Thank you

**Reply by carolandreazza** [reply]
Hello! 

I changed in the code what you indicated like this: 
ZepetoPlayers.instance.CreatePlayerWithZepetoId("", "jump97", spawninfo, true);

Following this example:
https://github.com/naverz/zepeto-script-sample/blob/main/Assets/2.%20Advanced/CharacterController/CharacterControllerSample.ts

But now I have this error:
![image](https://user-images.githubusercontent.com/29384128/158590438-3f63a6bc-6829-4013-b9c4-e2c1145ecb0c.png)

Now should I add the multiplayer code? I did not understand. Could you explain better?

**Reply by Zepeto-TechEvangelist** [reply]
Can you show me a screen capture of what it looks like when tested on your Unity Editor?
Also, it would be better to show the entire code of the character controller.

**Reply by carolandreazza** [reply]
Hi!
Follow:

Code of the character controller:
![image](https://user-images.githubusercontent.com/29384128/158854201-8bafe441-608a-4cc1-8024-13c5ddef74a0.png)

Screen capture:
![image](https://user-images.githubusercontent.com/29384128/158856483-cc625bbd-3351-443a-a4e6-65889108046f.png)

**Reply by Zepeto-TechEvangelist** [reply]
On the Unity editor, the login character is displayed well.
Did you build it and see if it works well on your mobile phone?
If there is no problem with the mobile phone test, ask for a review and ask for a detailed reason.

**Comment by Andwhy81kr** [comment]
Hi, I had been rejected at same issue.
and I find solution.

https://cafe.naver.com/zepetostudio/8

important things is "**WorldService.userId**".

and you create player at like this code.

ZepetoPlayers.instance.**CreatePlayerWithUserId**(WorldService.userId,  new Spawn(), true);

**Reply by carolandreazza** [reply]
Hi. I changed what you recommended and resubmitted. Thanks
